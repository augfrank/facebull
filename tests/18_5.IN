M0   C0   C3  2370
M1   C0   C4  2673
M2   C0   C5  2106
M3   C0   C6  2019
M4   C0   C10  2166
M5   C0   C11  2068
M6   C0   C12  2593
M7   C0   C13  2822
M8   C0   C15  2284
M9   C0   C16  2158
M10   C0   C17  2465
M11   C1   C0  2778
M12   C1   C2  2675
M13   C1   C3  2863
M14   C1   C4  2894
M15   C1   C5  2265
M16   C1   C6  2905
M17   C1   C7  2153
M18   C1   C10  2536
M19   C1   C13  2965
M20   C1   C15  2914
M21   C1   C16  2136
M22   C2   C1  2887
M23   C2   C8  2221
M24   C2   C10  2293
M25   C2   C11  2254
M26   C2   C13  2426
M27   C2   C17  2107
M28   C3   C0  2527
M29   C3   C5  2740
M30   C3   C6  2571
M31   C3   C7  2899
M32   C3   C8  2242
M33   C3   C9  2742
M34   C3   C10  2295
M35   C3   C11  2210
M36   C3   C12  2703
M37   C3   C14  2780
M38   C3   C15  2690
M39   C3   C16  2604
M40   C3   C17  2493
M41   C4   C1  2871
M42   C4   C2  2475
M43   C4   C6  2895
M44   C4   C7  2118
M45   C4   C12  2435
M46   C4   C13  2857
M47   C4   C14  2937
M48   C4   C16  2645
M49   C4   C17  2338
M50   C5   C0  2268
M51   C5   C4  2888
M52   C5   C6  2961
M53   C5   C7  2666
M54   C5   C8  2789
M55   C5   C9  2112
M56   C5   C12  2705
M57   C5   C16  2533
M58   C5   C17  2119
M59   C6   C0  2155
M60   C6   C2  2716
M61   C6   C4  2920
M62   C6   C5  2659
M63   C6   C8  2857
M64   C6   C9  2711
M65   C6   C11  2194
M66   C6   C13  2970
M67   C6   C14  2739
M68   C6   C17  2652
M69   C7   C1  2327
M70   C7   C2  2134
M71   C7   C4  2221
M72   C7   C5  2829
M73   C7   C6  2075
M74   C7   C11  2019
M75   C7   C12  2706
M76   C7   C13  2033
M77   C7   C14  2747
M78   C7   C15  2890
M79   C7   C16  2064
M80   C7   C17  2644
M81   C8   C1  2799
M82   C8   C5  2463
M83   C8   C7  2065
M84   C8   C9  2120
M85   C8   C10  2652
M86   C8   C11  2616
M87   C8   C13  2650
M88   C8   C14  2096
M89   C8   C16  2538
M90   C9   C1  2609
M91   C9   C3  2149
M92   C9   C4  2832
M93   C9   C5  2558
M94   C9   C7  2484
M95   C9   C10  2027
M96   C9   C11  2742
M97   C9   C12  2666
M98   C9   C14  2362
M99   C9   C15  2584
M100   C9   C17  2221
M101   C10   C0  2478
M102   C10   C1  2852
M103   C10   C2  2569
M104   C10   C5  2004
M105   C10   C6  2319
M106   C10   C7  2958
M107   C10   C11  2243
M108   C10   C12  2798
M109   C10   C13  2455
M110   C10   C14  2481
M111   C10   C15  2912
M112   C10   C17  2620
M113   C11   C2  2250
M114   C11   C3  2871
M115   C11   C4  2373
M116   C11   C5  2161
M117   C11   C7  2661
M118   C11   C9  2047
M119   C11   C10  2782
M120   C11   C12  2099
M121   C11   C15  2087
M122   C12   C5  2123
M123   C12   C6  2683
M124   C12   C7  2954
M125   C12   C8  2204
M126   C12   C9  2150
M127   C12   C11  2603
M128   C12   C13  2285
M129   C12   C14  2820
M130   C12   C16  2235
M131   C13   C1  2178
M132   C13   C2  2454
M133   C13   C4  2139
M134   C13   C5  2608
M135   C13   C9  2939
M136   C13   C10  2426
M137   C13   C16  2458
M138   C13   C17  2806
M139   C14   C0  2959
M140   C14   C2  2214
M141   C14   C3  2759
M142   C14   C5  2958
M143   C14   C7  2524
M144   C14   C8  2723
M145   C14   C12  2702
M146   C14   C15  2470
M147   C14   C16  2161
M148   C15   C1  2762
M149   C15   C2  2426
M150   C15   C3  2033
M151   C15   C7  2284
M152   C15   C11  2183
M153   C15   C12  2368
M154   C15   C13  2848
M155   C15   C16  2670
M156   C16   C0  2916
M157   C16   C3  2806
M158   C16   C5  2960
M159   C16   C6  2815
M160   C16   C7  2504
M161   C16   C8  2189
M162   C16   C9  2543
M163   C16   C11  2487
M164   C16   C13  2324
M165   C16   C14  2577
M166   C16   C15  2580
M167   C16   C17  2318
M168   C17   C1  2310
M169   C17   C2  2621
M170   C17   C4  2765
M171   C17   C5  2160
M172   C17   C6  2288
M173   C17   C8  2971
M174   C17   C11  2242
M175   C17   C12  2721
M176   C17   C15  2908
M177   C17   C16  2227
