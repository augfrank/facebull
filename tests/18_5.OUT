38779
8 21 25 32 44 58 59 73 88 95 104 120 126 133 140 150 164 168
